praw/models/reddit/subreddit.py

```diff
@@ -3827,6 +3827,21 @@ def delete_banner_hover_image(self):
         data = {"secondaryBannerPositionedImage": ""}
         self._update_structured_styles(data)
 
+    def delete_mobile_banner(self):
```

Comment:
Could you move this down (right before `delete_mobile_header` I believe) so they are sorted alphabetically please?